examples/vkcs-compute-instance-create/base.tf

```diff
@@ -0,0 +1,20 @@
+data "vkcs_compute_flavor" "compute" {
```

Comment:
if we can use the same names of base objects? this would be useful for guides. lets discuss